pkg/network/usm/kafka_monitor_test.go

```diff
@@ -1349,17 +1412,23 @@ func testKafkaProduceRaw(t *testing.T, tls bool, apiVersion int) {
 				cleanProtocolMaps(t, "kafka", monitor.ebpfProgram.Manager.Manager)
 			})
 			req := tt.buildRequest(tt.topic)
-			formatter := kmsg.NewRequestFormatter(kmsg.FormatterClientID("kgo"))
-			data := formatter.AppendRequest(make([]byte, 0), &req, int32(99))
-			msgs := []Message{{request: data}}
+			var msgs []Message
```

Comment:
Should we add "split" tests for produce also like fetch so that we test that packet break handling is correct?

Reply:
Yes, will do

Reply:
Done